src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppElasticBeanstalk.recipe

```diff
@@ -329,6 +329,21 @@
                     ]
                 }
             ]
+        },
+        {
+            "Id": "HealthCheckURL",
+            "Name": "Health Check URL",
+            "Description": "Customize the load balancer health check to ensure that your application, and not just the web server, is in a good state",
```

Comment:
Can you update the description to reflect that this OptionSettingItem is only required if the environment type is load balanced?

Reply:
Since the option will only be displayed if the environment type is load balanced I don't think we need to make the description extra wordy.

Reply:
I agree with Norm. This is only displayed when the env is load balanced.